src/redux/reducers/TodoReducer.ts

```diff
@@ -53,6 +54,21 @@ export const todoReducer = (
         return item;
       });
 
+      return {
+        ...state,
+        todoItems: newTodoItems,
+      };
+    }
+    case TodoActionTypes.EDIT_ITEM: {
+      const { id, text }: EditItemPayload = action.payload;
+
+      const newTodoItems = state.todoItems.map(item => {
+        if (item.id === id) {
+          return { ...item, text };
```

Comment:
does your editing logic allow change only text? why I as user can't change date?

Reply:
why did you close it?! and haven't leave any message or fixed my remarks?